src/TropicalGeometry/TropicalPolyhedron/properties.jl

```diff
@@ -0,0 +1,368 @@
+2-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (-1), (0)]
+ [(0), (-4), (-1)]
+
+julia> TT = tropical_semiring();
+
+julia> Q = tropical_convex_hull(TT[0 1 2; inf 0 3; inf inf 0; 0 1 0]);
+
+julia> vertices(Q)
+3-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (1), (2)]
+ [infty, (0), (3)]
+ [infty, infty, (0)]
+```
+"""
+function vertices(as::Type{PointVector{T}}, P::TropicalPolyhedron) where {T<:TropicalSemiringElem}
+  n = n_vertices(P)
+
+  return SubObjectIterator{as}(P, _vertex_of_polyhedron, n)
+end
+vertices(P::TropicalPolyhedron{M}) where {M<:MinOrMax} = vertices(PointVector{TropicalSemiringElem{M}}, P)
+
+@doc raw"""
+    n_vertices(P::TropicalPolyhedron)
+
+Return the number of vertices of `P`.
+"""
+n_vertices(P::TropicalPolyhedron) = nrows(pm_object(P).VERTICES::AbstractMatrix)
+
+function _vertex_of_polyhedron(::Type{PointVector{TropicalSemiringElem{M}}}, P::TropicalPolyhedron, i::Int) where {M<:MinOrMax}
+  T = tropical_semiring(convention(P))
+
+  return point_vector(
+    T,
+    T.(P.pm_tpolytope.VERTICES[i,:])
+  )
+end
+
+@doc raw"""
+    pseudovertices([as::Type{T} = PointVector,] P::TropicalPolyhedron)
+
+Return an iterator over the pseudovertices of `P` in the tropical projective torus, 
+that is the zero-dimensional cells in the covector decomposition of `P`.
+
+# Examples
+The following retrievs the pseudovertices of a bounded tropical polytope:
+```jldoctest
+julia> P = tropical_convex_hull(QQ[0 -1 0; 0 -4 -1]);
+
+julia> vertices(P)
+2-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (-1), (0)]
+ [(0), (-4), (-1)]
+
+julia> pseudovertices(P)
+3-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (-1), (0)]
+ [(0), (-4), (-1)]
+ [(0), (-3), (0)]
+```
+
+For an unbounded tropical polytope, this only outputs the points for which all coordinates are finite:
+```jldoctest
+julia> TT = tropical_semiring();
+
+julia> P = tropical_convex_hull(TT[0 1 4; inf 0 2; inf inf 0]);
+
+julia> pseudovertices(P)
+2-element SubObjectIterator{PointVector{TropicalSemiringElem{typeof(min)}}}:
+ [(0), (1), (4)]
+ [(0), (1), (3)]
+```
+"""
+function pseudovertices(as::Type{PointVector{T}}, P::TropicalPolyhedron) where {T<:TropicalSemiringElem}
+  CT = pm_object(P).PSEUDOVERTEX_COARSE_COVECTORS::Polymake.Matrix{<:Integer}
+  n = count(!iszero, eachrow(CT))
+  TT = tropical_semiring(convention(P))
+
+  return SubObjectIterator{as}(
+    P,_pseudovertices,n
+  )
+end
+
+function pseudovertices(as::Type{PointVector{T}}, P::TropicalPointConfiguration) where {T<:TropicalSemiringElem}
+  n = n_pseudovertices(P)
+
+  return SubObjectIterator{as}(P, _pseudovertices, n)
+end
+pseudovertices(P::Union{TropicalPolyhedron{M},TropicalPointConfiguration{M}}) where {M<:MinOrMax} = pseudovertices(PointVector{TropicalSemiringElem{M}}, P)
+
+n_pseudovertices(P::TropicalPointConfiguration) = pm_object(P).PSEUDOVERTICES |> nrows
```

Comment:
This is not meant as a blocker to this PR, it could be removed in a follow-up, but I want to record this point of view here:

I'd greatly prefer if this code did not use the pipeline operator `|>` -- it's just yet another Julia feature that's cool if you know it (I personally like it!), and it's great for the REPL -- but many people writing Julia code will still not be familiar with it, raising the barrier for working on this part of the code.

In fact I just checked, and this operator is used just 4 times in all of Oscar.jl -- twice in `experimental`, and twice in `src`.